crates/nu-protocol/src/errors/shell_error.rs

```diff
@@ -1557,12 +1557,15 @@ impl From<Spanned<io::Error>> for ShellError {
                     msg: "unknown error".into(),
                     span,
                 },
-            }
-        } else {
-            Self::IOErrorSpanned {
+            },
+            io::ErrorKind::TimedOut => Self::NetworkFailure {
```

Comment:
Wondering if we ever might run into a non-networking timeout on some obscure file system?